core/arch/opnd.h

```diff
@@ -2843,7 +2843,31 @@ DR_API
 bool
 reg_is_stolen(reg_id_t reg);
 
-/* internal version */
+/* Internal version */
```

Comment:
It is not clear what this comment applies to (singular; separated from everything; if you want to apply it to a group of functions, use banner comments to clearly group them...).  I would remove this and move all the stuff below into decode_private.h.  That also removes the need for any ifdefs.

Reply:
Done (I've moved the definitions to x86/decode.c)